cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -798,9 +799,11 @@ private boolean getProposals(HttpServletRequest request, HttpServletResponse res
       out.write(String.format("%n%nOptimized load:").getBytes(StandardCharsets.UTF_8));
       out.write(loadAfterOptimization.getBytes(StandardCharsets.UTF_8));
     } else {
+      Map<String, Object> proposalMap = new HashMap<>();
       if (!verbose) {
-        out.write(
-            KafkaCruiseControlServletUtils.getProposalSummary(optimizerResult).getBytes(StandardCharsets.UTF_8));
+        proposalMap.put("proposalSummary", KafkaCruiseControlServletUtils.getProposalSummary(optimizerResult, true));
```

Comment:
If we pass `json` rather than an explicit `true` flag, would it make the code more clear?

-- Based on looking at the rest of the code, would it be more clear to use separate function in case it is not `json` (return `String`), and it is `json` (return `Map<String, Object>`).